packages/terra-notification-dialog/src/NotificationDialog.scss

```diff
@@ -0,0 +1,99 @@
+:local {
+  .notification-dialog {
+    background: var(--notification-dialog-background, #fff);
+    border-radius: var(--notification-dialog-border-radius, 0.3125rem);
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+
+    @media screen and (max-width: 768px) {
+      width: calc(100% - 2 * var(--notification-dialog-width, 10px));
+    }
+  }
+
+  .notification-dialog-body {
+    background-color: var(--notification-dialog-body-color, #fff);
+    display: inline-flex;
+    flex: 1 1 auto;
+    flex-direction: initial;
```

Comment:
Using `initial` as a value is not supported in IE 10 or IE 11. We should replace this with either `column` or `row`, whatever the intended design requires.